zulip.md

```diff
@@ -0,0 +1,60 @@
+# Zulip
+
+##  Getting started on Zulip:
+
+### If you’re new here, here’s what you can do to get started with this community:
+
+1.  :heavy_check_mark:  [Introduce yourself on #newcomers in the #newcomers > Introductions topic and tell us your motivations for joining so we can get to know you and help you on your journey](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers)
+
+2.  :calendar:  [Follow our Open Sessions in Calendar Events topic on #announcements or through our AnitaB.org Open Source calendar (PST:clock1:). These are voice calls you can participate in and get to know a bit more our community and projects](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles)
+
+##   Learning Resources :eyeglasses:
+### To keep this community safe for you and others.
+
+1.  [Code of Conduct](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/code_of_conduct.md)     
+
+2.  [Reporting Guidelines](https://github.com/anitab-org/anitab-org.github.io/blob/develop/docs/reporting_guidelines.md) 
+
+3.  [To get Acquainted with Zulip](https://zulipchat.com/help/getting-started-with-zulip)  
+
+##  Streams:point_down:
+    
+1. Category baseds: These are streams based on a specific type of contribution (e.g.: quality assurance, coding, documentation, etc). Join streams of your interest such as:
```

Comment:
can you make these lists non enumerated? So instead of using 1, 2, 3, using "*" or "-" to define each entry. This is because there is no streams more important then other, they are all at the same relevance level

Reply:
Definitely, working on it right now.